### PR TITLE
Rename variables names from dimX to columns and dimY to rows

### PR DESCRIPTION
**Pull request by rafaelgmota**
This patch makes uniform the use of dimensions and coordinates to specify network topology avoiding confusion.
#81 was caused by a misunderstood related with the issue above.

### CODE REVIEW

File: src/rsbr.java [hunk 1]
```diff
@@ -12,25 +12,25 @@ public static void main(String[] args) {
         String topologyFile = null;
         String volumePath = null;
         boolean shouldMerge = true;
-        int dim = 4;
-        int dimX = dim;
-        int dimY = dim;
+        //int dim = 4;
```
Comment (jmferreiratech) [inline src/rsbr.java]: It wasn't necessary to change these variables values.

**Review by jmferreiratech — CHANGES_REQUESTED** [review]
There'are more uses of 'dimX' and 'dimY' through the code, please get ride of them all once for all.

**Comment by jmferreiratech** [comment]
Take the chance to fix the typo at the commit message.

**Comment by rafaelgmota** [comment]
Modifications done, please verify if there is more errors to correct

**Review by jmferreiratech — APPROVED** [review]

**Review by rmilfont — APPROVED** [review]

**Pull request merged**